src/platform/plugins/shared/dashboard/server/index.ts

```diff
@@ -8,13 +8,29 @@
  */
 
 import { PluginInitializerContext, PluginConfigDescriptor } from '@kbn/core/server';
-import { configSchema, ConfigSchema } from './config';
+import { schema, TypeOf } from '@kbn/config-schema';
 
-export const config: PluginConfigDescriptor<ConfigSchema> = {
+export const configSchema = schema.object({
+  allowByValueEmbeddables: schema.boolean({ defaultValue: true }),
+});
+
+export const config: PluginConfigDescriptor<TypeOf<typeof configSchema>> = {
   exposeToBrowser: {
```

Comment:
Does `allowByValueEmbeddables` still need to be exposed to browser if its no longer used any where. I think we could remove `exposeToBrowser` section.

Reply:
done in [d20db46](https://github.com/elastic/kibana/pull/221165/commits/d20db46402bb64276c47fafb869865e03efce5bb)